CONTRIBUTING.md

```diff
@@ -1,26 +1,101 @@
-# Dynatrace Operator
+- [Contributing](#contributing)
+  - [How to run unit tests](#how-to-run-unit-tests)
```

Comment:
```suggestion
  - [Unit tests](#how-to-run-unit-tests)
```
I prefer to name it more general, so we can add more items later regarding unit-tests. For example: how to debug unit tests etc.

Reply:
Done :+1: